app/components/channel_list/header/plus_menu/index.tsx

```diff
@@ -7,6 +7,7 @@ import {useIntl} from 'react-intl';
 import CompassIcon from '@components/compass_icon';
 import {Screens} from '@constants';
 import {useTheme} from '@context/theme';
+import {makeCloseButton as makeCreateOrEditCloseButton} from '@screens/create_or_edit_channel/create_or_edit_channel';
```

Comment:
seems a bit weird.. 0/5 if you want to move the function to another file

Reply:
Now that I think more about it, on the first render we will be setting the button either way, so I will not set the button during creation. That should remove the current weirdness.